### PR TITLE
Azure Database for PostgreSQL doc and old doc location tag fix

### PR DESCRIPTION
**Pull request by sachin-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-cosmos-db.md [hunk 1]
```diff
@@ -148,12 +149,12 @@ Currently, only Azure Cosmos DB for NoSQL database account type supports exporti
 In this section, you will configure a pipeline for shipping diagnostic logs from Azure Monitor to an Event Hub.
 
 1. To set up the Azure Event Hubs source in Sumo Logic, refer to [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
-1. If you want to audit Azure Cosmos DB control plane operations, [disable the key based metadata write access](https://learn.microsoft.com/en-us/azure/cosmos-db/audit-control-plane-logs#disable-key-based-metadata-write-access).
-1. To create the diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/cosmos-db/monitor-resource-logs?tabs=azure-portal#create-diagnostic-settings). Perform the following steps for each Azure Cosmos DB account that you want to monitor.
+2. If you want to audit Azure Cosmos DB control plane operations, [disable the key based metadata write access](https://learn.microsoft.com/en-us/azure/cosmos-db/audit-control-plane-logs#disable-key-based-metadata-write-access).
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-cosmos-db.md:152]: no need to change numbering in future in markdown it automatically does the numbering if one puts `1.`

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 2]
```diff
@@ -24,37 +24,208 @@ For Azure Database for PostgreSQL, you can collect the following logs and metric
...
+   ```
+
+### Configure metric rules
+
+Create the following metrics rules by following the instructions in [Create a metrics rule](/docs/metrics/metric-rules-editor/#create-a-metrics-rule).
+
+#### Azure observability metadata extraction flexible postgresql server level
+
+```sql
+Rule Name: AzureObservabilityMetadataExtractionAzureDatabaseForPostgreSQLLevel
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:100]: resourceId=/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/MICROSOFT. DBFORPOSTGRESQL/FLEXIBLESERVERS/*

replace $resourceId._3  with MICROSOFT. DBFORPOSTGRESQL
relace $resourceId._4 with  $resourceId._3

need to do the same for azure database for mysql

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 3]
```diff
@@ -24,37 +24,208 @@ For Azure Database for PostgreSQL, you can collect the following logs and metric
...
+3. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Flexible PostgreSQL Server resource that you want to monitor.
+   1. Choose `Stream to an event hub` as destination.
+   1. Select `AllMetrics`.
+   1. Use the Event hub namespace created by the ARM template in Step 2 above. You can create a new Event hub or use the one created by ARM template. You can use the default policy `RootManageSharedAccessKey` as the policy name. <br/><img src={useBaseUrl('img/send-data/azureflexible-postgresqlserver-metrics.png')} alt="Azure flexible postgresql server metrics" style={{border: '1px solid gray'}} width="800" />
+1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Redis Cache Tag Location" style={{border: '1px solid gray'}} width="400" />
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:127]: alt text for all of them should be different here it should be Azure Database for Postgresql

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 4]
```diff
@@ -24,37 +24,208 @@ For Azure Database for PostgreSQL, you can collect the following logs and metric
...
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Skip this step if you are already collecting activity logs for a subscription.
+
+:::note
+Since this source contains logs from multiple regions, ensure that you do not tag this source with the location tag.
+:::
+
+## Installing the Azure Flexible Database for PostgreSql app
+
+Now that you have set up data collection, install the Azure Load Balancer Sumo Logic app to use the pre-configured dashboards that provide visibility into your environment for real-time analysis of overall usage.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:162]: Azure Load Balancer -> Azure Database for PostgreSQL

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 5]
```diff
@@ -24,37 +24,208 @@ For Azure Database for PostgreSQL, you can collect the following logs and metric
...
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureDatabaseForPostgresql/Azure-Database-for-PostgreSQL-Administrative-Operations.png')} alt="Azure Database for PostgreSql - Administrative Operations" style={{border: '1px solid gray'}} width="800" />
+
+### Autovacuum
+**Azure Database for PostgreSQL - Autovacuum** dashboard provides details like Estimated Dead Rows User Tables, Estimated Live Rows User Tables, Estimated Modifications User Tables, Analyze Counter User Tables, Vacuum Counter User Tables, User Tables Vacuumed vs AutoVacuumed, User Tables Analyzed vs User Tables Auto etc.
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureDatabaseForPostgresql/Azure-Database-for-PostgreSQL-Autovacuum.png')} alt="Azure Database for PostgreSql - Autovacuum" style={{border: '1px solid gray'}} width="800" />
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:184]: the bucket name should be sumologic-app-data-v2 in S3.
Need to check for other apps as well. 
sumologic-app-data bucket contains deprecated data.

**Review by himanshu219 — CHANGES_REQUESTED** [review]
minor changes needed, the doc format should follow. You can use chatgpt to upload the screenshot and get the bulleted points summary for all the dashboards. 
Use dashboard to (bullet format)
1.
2.

In Azure Database for MySQL and Azure Application Gateway also we missed following this convention, in rest of the azure apps we are following it.

**Review by jpipkin1 — APPROVED** [review]
@sachin-sumologic, your updates look very good! I have no corrections. 

By the way, you can start each item in a numbered list with "1." because they will be numbered automatically. See our [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#numbered-lists).

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**